### PR TITLE
WoT Week Planning Folder

### PR DESCRIPTION
**Pull request by egekorkan**
Moving over content from https://www.w3.org/WoT/IG/wiki/WG_WoT_Thing_Description_WebConf#July_4_2024

### CODE REVIEW

File: events/2024.11.Munich/Readme.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# WoT Week at Munich
+
+The logistics can be found at https://www.w3.org/WoT/IG/wiki/Wiki_for_WoT_Week_2024_planning .
```
Comment (danielpeintner) [inline events/2024.11.Munich/Readme.md:3]:
```suggestion
The logistics can be found at https://www.w3.org/WoT/IG/wiki/Wiki_for_WoT_Week_2024_planning.
```

File: events/2024.11.Munich/Readme.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+## WIP - TD Topics
+
+### Making Sure
+
+* Additional Responses
+  * Error Handling on the application level (Scripting API is connected here)
+  * Multiple possible successful responses
+  * Default behavior
+* Linking between TDs
+* Specifying multiple types (at types) to TDs: This is confusing to people. We need more guidance.
```
Comment (danielpeintner) [inline events/2024.11.Munich/Readme.md:24]: Not sure what is meant with "(at types)" ?

File: events/2024.11.Munich/Readme.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* Complex actions: Async and sync
+* Meta Operations (top-level form operations):Are they implemented correctly
```
Comment (danielpeintner) [inline events/2024.11.Munich/Readme.md:26]:
```suggestion
* Meta Operations (top-level form operations): Are they implemented correctly
```

**Comment by egekorkan** [comment]
Failing test is irrelevant at this stage but should be fixed before TD submissions start.

**Review by danielpeintner — COMMENTED** [review]
Some early comments below...

File: events/2024.11.Munich/Readme.md (hunk 2 above)
Comment (egekorkan) [reply events/2024.11.Munich/Readme.md:24]:
```suggestion
* Specifying multiple `@type`s to TDs: This is confusing to people. We need more guidance.
```

**Comment by egekorkan** [comment]
> Some early comments below...

@danielpeintner Applied your feedback

**Pull request merged**